module/Kanbanize/src/Kanbanize/Service/ImportDirector.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+namespace Kanbanize\Service;
+
+use Application\Entity\User;
+use Application\Service\UserService;
+use Kanbanize\KanbanizeStream;
+use Kanbanize\KanbanizeTask;
+use People\Organization;
+use Prooph\EventStore\EventStore;
+use Rhumsaa\Uuid\Uuid;
+use TaskManagement\Service\StreamService;
+use TaskManagement\Service\TaskService;
+use TaskManagement\Entity\Task as ReadModelTask;
+use TaskManagement\Entity\TaskMember;
+use People\Service\OrganizationService;
+
+
+
+class ImportDirector {
+	
+	/**
+	 * @var KanbanizeService
+	 */
+	protected $kanbanizeService;
+	/**
+	 * @var TaskService
+	 */
+	protected $taskService;
+	/**
+	 * @var StreamService
+	 */
+	protected $streamService;
+	/**
+	 * @var EventStore
+	 */
+	protected $transactionManager;
+	/**
+	 * @var UserService
+	 */
+	protected $userService;
+	/**
+	 * @var OrganizationService
+	 */
+	protected $organizationService;
+	
+	private $apiKey;
+
+	public function __construct(KanbanizeService $kanbanizeService,
+			TaskService $taskService,
+			StreamService $streamService,
+			EventStore $transactionManager,
+			UserService $userService,
+			OrganizationService $organizationService){
+		$this->kanbanizeService = $kanbanizeService;
+		$this->taskService = $taskService;
+		$this->streamService = $streamService;
+		$this->transactionManager = $transactionManager;
+		$this->userService = $userService;
+		$this->organizationService = $organizationService;
+	}
+	/**
+	 * 
+	 * @param Organization $organization
+	 * @param User $requestedBy
+	 */
+	public function import(Organization $organization, User $requestedBy){
+		$api = $this->initApi($organization, $this->apiKey);
```

Comment:
Ho scelto di passare l'oggetto $api nel costruttore dell'Importer in modo da poterlo mockare più agevolmente nei test